### PR TITLE
DOC-2366: Add revision history diff API

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Site: [link - #diff](http://docs-feature-71-doc-2366.staging.tiny.cloud/docs/tinymce/latest/revisionhistory/#diff)

Description of changes:
- Documented diff() API of the Revision history doc
- Removed orphan `accordion_apis.adoc` and cleaned up

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/partials/plugin-apis/ai-apis.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ tinymce.activeEditor.plugins.ai.getThreadLog();
 [[getThreadLog]]
 === The `getThreadLog` API
 
-A user or integrator can retrieve the history of each conversation thread by calling `+editor.ai.getThreadLog()+` on an editor instance with the {pluginname} plugin enabled.
+A user or integrator can retrieve the history of each conversation thread by calling `+editor.ai.getThreadLog()+` on an editor instance with the {pluginname} plugin enabled.
```
Comment (MitchC1999) [inline modules/ROOT/partials/plugin-apis/ai-apis.adoc:17]:
```suggestion
A user or integrator can retrieve the history of each conversation thread by calling `+editor.ai.getThreadLog()+` on an editor instance with the {pluginname} plugin enabled.
```
Not sure why this has been modified

File: modules/ROOT/partials/plugin-apis/revisionhistory-apis.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+----
+const input1 = `<p>The banner is <span style="color: rgb(224, 62, 45);">red</span></p>`;
+const input2 = `<p>The flag is <span style="color: rgb(22, 145, 121);">red</span></p>`;
+tinymce.activeEditor.plugins.revisionhistory.diff(input1, input2);
+/*
+Expected output:
+<p>The
+ <del class="tox-revisionhistory__annotation--removed">banner</del><ins class="tox-revisionhistory__annotation--added">flag</ins>
+ is 
+<ins class="tox-revisionhistory__annotation--modified"><span style="color: rgb(22, 145, 121);">red</span></ins>
```
Comment (MitchC1999) [inline modules/ROOT/partials/plugin-apis/revisionhistory-apis.adoc:23]: Should this have been prefixed with `tox`? It's not part of the UI

**Review by MitchC1999 — APPROVED** [review]

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/partials/plugin-apis/ai-apis.adoc (hunk 1 above)
Comment (tiny-ben-tran) [reply modules/ROOT/partials/plugin-apis/ai-apis.adoc:17]: My bad. It is a mistake

File: modules/ROOT/partials/plugin-apis/revisionhistory-apis.adoc (hunk 2 above)
Comment (tiny-ben-tran) [reply modules/ROOT/partials/plugin-apis/revisionhistory-apis.adoc:23]: Not sure but it is definitely not part of the UI. I am not aware of rules for CSS class names that are not part of the UI

**Pull request merged**